app/views/documents/show/_content_settings.html.erb

```diff
@@ -0,0 +1,24 @@
+<% backdating_status = capture do %>
+  <% if @edition.backdated_to.present? %>
+    <% @edition.backdated_to.strftime("%-d %B %Y") %>
```

Comment:
Another minor thing - do you think we could re-use the (suitably renamed) SchedulingHelper here? Possibly something for a follow-up PR, though!

Reply:
I've added this is to 'Things we need to do' on the card.